### DISCUSSION TITLE
container/summary

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#179] [General]
# container/summary

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/container/summary.html

**Comment by xiaozheng-sun** [comment]
让 Docker 大行其道的潜能是 cgroups？或者是 namespace？再或者是镜像和联合文件系统？ 促使 Docker 引领容器技术潮流的潜能是 cgroup 么？或者是 namespace？ 再或者是镜像和联合文件系统？

重复了，bro

**Reply by isno** [reply]
谢谢 bro。我正在逐章修正，您有兴趣可以看看前四章。容器这一章估计在5月初修正完。

**Comment by mogita** [comment]
图里有个 typo，7.5 的第三项 Volum 或许应该是 Volume？

**Reply by isno** [reply]
谢谢，图里面写错了，我后面更新一下。
